<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/Resource/Cal.ico</normaloff>:/Resource/Cal.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{

background-color: #000000;

border: 1px solid gray;

}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="4">
     <widget class="QLineEdit" name="Display">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{

background-color: gray;
border: 1px solid gray;
color: #FFFFFF


}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="maxLength">
       <number>1000000000</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLineEdit" name="DisplayAux">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{

background-color: black;
border: 1px solid gray;
color: #FFFFFF


}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #00aaff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="shortcut">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QPushButton" name="pushButton_Square">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}</string>
      </property>
      <property name="text">
       <string>x²</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="pushButton_Fact">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}</string>
      </property>
      <property name="text">
       <string>n!</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QPushButton" name="pushButton_CLOne">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #ff0000;
border: 1px solid gray;
padding: 3px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resource.qrc">
        <normaloff>:/Resource/Del.ico</normaloff>:/Resource/Del.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="shortcut">
       <string>Backspace</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="pushButton_SquareRoot">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}</string>
      </property>
      <property name="text">
       <string>√</string>
      </property>
     </widget>
    </item>
    <item row="17" column="3">
     <widget class="QPushButton" name="pushButton_Equals">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #00aaff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="shortcut">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_Mod">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}</string>
      </property>
      <property name="text">
       <string>|x|</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_Sen">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #aaffff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>sen</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pushButton_Div">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #ffaa00;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>÷</string>
      </property>
      <property name="shortcut">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="4">
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{

background-color: black;
border: 1px solid gray;
color: #FFFFFF


}</string>
      </property>
      <property name="text">
       <string>GilSoft</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="17" column="2">
     <widget class="QPushButton" name="pushButton_Point">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="shortcut">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #00aaff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="shortcut">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_Tan">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #aaffff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #00aaff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="shortcut">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="pushButton_Mul">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #ffaa00;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>×</string>
      </property>
      <property name="shortcut">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="pushButton_Ln">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #aaffff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #00aaff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="shortcut">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #00aaff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="shortcut">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QPushButton" name="pushButton_AC">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #ff0000;
border: 1px solid gray;
padding: 3px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}</string>
      </property>
      <property name="text">
       <string>&amp;AC</string>
      </property>
      <property name="shortcut">
       <string>Del</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pushButton_Exp">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #aaffff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>exp</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_ChangeSign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #C0C0C0;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #00aaff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="shortcut">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #00aaff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="shortcut">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="pushButton_Sub">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #ffaa00;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="shortcut">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButton_Add">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #ffaa00;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="shortcut">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #00aaff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="shortcut">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #00aaff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="shortcut">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_Cos">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{

background-color: #aaffff;
border: 1px solid gray;
padding: 5px;


}
QPushButton:pressed
{

background-color: #A9A9A9;
border: 1px solid gray;
padding: 5px;


}
</string>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>395</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
